src/EFCore.PG/Metadata/INpgsqlRelationalAnnotations.cs

```diff
@@ -0,0 +1,23 @@
+using System.Collections.Generic;
+using JetBrains.Annotations;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Metadata
+{
+    /// <summary>
+    /// Represents relational-specific annotations specific to Npgsql.
+    /// </summary>
+    public interface INpgsqlRelationalAnnotations
```

Comment:
Any reason these don't simply belong in `INpgsqlModelAnnotations`? I don't see any sense in distinguishing between Npgsql non-relational and relational annotations, as you seem to be doing - Npgsql is a relational provider...